tests/unit/dom/fill-in-test.js

```diff
@@ -44,6 +44,17 @@ module('DOM Helper: fillIn', function(hooks) {
     );
   });
 
+  test('filling in a disabled element', async function(assert) {
+    element = buildInstrumentedElement('input');
+    element.setAttribute('disabled', true);
+
+    await setupContext(context);
+    assert.rejects(
+      fillIn(`#${element.id}`, 'foo'),
+      new Error('Can not `fillIn` disabled [object HTMLInputElement]')
```

Comment:
why do we have `[object HTMLInputElement]` in here?

Reply:
we are passing an element instace, which is not focusable, into throw statement. Should prbably change it to use passed specifier(selector or Element) instead. Will update, thanks!